ex02.ex

```diff
@@ -53,7 +53,9 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
+  first2equal = fn([h|t]) ->
```

Comment:
# -3

This code will actuall accept a single element list, and then blow up trying to do hd([]).

What I was looking for was

~~~ elixir
def first2equal([h, h | _t]), do: true
def first2equal(_), do: false
~~~

or

~~~ elixir
def first2equal([h1, h2 |_ t]), do: h1 == h2
~~~